### PR TITLE
NCC alloy v2 updates

### PR DESCRIPTION
**Pull request by struan**
For mysociety/commercial-staging#1932

### CODE REVIEW

File: t/open311/endpoint/alloyv2_ncc.yml [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+  # method put these values in the attributes, which would require
+  # FMS to know about specific Alloy attribute IDs, handle this mapping
+  # on this end.
+  # NB: This will override any resource_attribute_defaults if present.
+  # TODO: These currently apply to all source types/designs.
+  "request_to_resource_attribute_mapping": {
+    "title": 'attributes_enquiryInspectionRFS1001181Summary1009859_5d3245d4fe2ad806f8dfbb15', # summary
+    "description": 'attributes_enquiryInspectionRFS1001181Explanation1009860_5d3245d5fe2ad806f8dfbb1a', # explanation
+    "fixmystreet_id": 'attributes_enquiryInspectionRFS1001181SourceID1009855_5d3245d1fe2ad806f8dfbb06', # Source ID
+    #"category": 'attributes_enquiryInspectionRFS1001181Category1011685_5d3245dbfe2ad806f8dfbb33', # category
```
Comment (davea) [inline t/open311/endpoint/alloyv2_ncc.yml:65]: There are a lot of commented out lines in this config, can they be removed? This file is also quite big in general, is it worth trimming it down to only what's required for the tests?

File: t/open311/endpoint/northamptonshire_alloyv2.t [hunk 2]
```diff
@@ -290,5 +294,55 @@ This is an update"
         } ], 'correct json returned';
 };
 
+subtest "check fetch problem" => sub {
+    set_fixed_time('2014-01-01T12:00:00Z');
```
[the remark this answers is not in the document]
Comment (struan) [reply t/open311/endpoint/northamptonshire_alloyv2.t:307]: it's required for all the problem tests. however I was never calling `restore_time` so as long as you left it in for the first one it was all fine. I've put in `restore_time` calls and removed the `set_fixed_time`ones where they're not required.

File: t/open311/endpoint/alloyv2_ncc.yml (hunk 1 above)
Comment (struan) [reply t/open311/endpoint/alloyv2_ncc.yml:65]: good shout

**Comment by struan** [comment]
superseded by https://github.com/mysociety/open311-adapter/pull/140

**Pull request closed without merging**